box.json

```diff
@@ -1,19 +1,10 @@
 {
-    "chmod": "0755",
-    "directories": ["src"],
     "files": [
         "LICENSE",
         "i18n.php"
     ],
-    "finder": [
-        {
-            "name": ["*.php", "*.xsd", "LICENSE"],
-            "exclude": ["Tests", "tests", "sebastian", "phpunit", "phpspec", "process", "filesystem"],
-            "in": "vendor"
-        }
+    "compactors": [
+        "KevinGH\\Box\\Compactor\\Php"
```

Comment:
Is there a TL;DR somewhere of what difference this makes to the other compactor? Or are they both just minifying classes and we should just use it?

Reply:
They are both just minifying the PHP-Code (stripping code-comments basically). And both are part of the respective package. As this now uses `box` and no longer `box2` the classes are named differently but in essence still do the same.